PORTAL/jobs/__init__.py

```diff
@@ -71,11 +80,7 @@ def __init__(self, root: Optional[str] = None):
 
         super().__init__(root)
 
-        self.jobs = _common.JobsFS(self.root)
-        self.jobs.context = 'portal'
```

Comment:
There were a couple of places in #108 where I was also copying this instance member around to make things work that will probably no longer by necessary as well.  (Just a note for myself to remove that and retest after rebasing on this).